experimental/framework/fluid-static/src/containerCode.ts

```diff
@@ -115,6 +122,25 @@ export class RootDataObject
         super.dispose();
     }
 
+    public setContainer(container: Container) {
+        this._container = container;
+    }
+
+    public close() {
```

Comment:
My preference would be to omit `close()` and `closed`, and instead just add a `disposed` to match `dispose()`.  Having two methods that do basically the same thing is confusing.

Reply:
close and closed align with the existing Container API. 

Also dispose is already a public function implemented and used by both PureDataObject and EventForwarder so the change to expose these would be much larger.

We could always just create a custom DO and not use Aqueduct but that seems pretty heavy weight.

This is an interesting inflection because we've hit a scenario where the FluidContainer API and the Aqueduct DO API intersect.

Reply:
I was talking to Micah this morning about the collision problem - a good option might be to write FluidContainer as a class (exported from fluid-static and consumed by the client packages) which composes the root data object and the container.  So the client package would do something like

```typescript
return new FluidContainer(rootDataObject, container);
```

And that would get us a clean interface to work with while still using Aqueduct.  Plus they wouldn't have to make a .setContainer() call and we could avoid having to do the undefined asserts, etc.

Reply:
+1 to this. It would also allow the FluidContainer to continue to expose additional capabilities, as required, from the container itself that don't have any service-dependencies